tests/cassettes/test_dd/test_financial_model/test_rating.yaml

```diff
@@ -36,4 +36,82 @@ interactions:
     status:
       code: 200
       message: ''
+- request:
+    body: null
+    headers:
+      Connection:
+      - close
+      Host:
+      - financialmodelingprep.com
+      User-Agent:
+      - Python-urllib/3.8
+    method: GET
+    uri: https://financialmodelingprep.com/api/v3/historical-rating/PLTR?apikey=845b22316eeec4168d249976bba94330
+  response:
+    body:
+      string: "[ {\n  \"symbol\" : \"PLTR\",\n  \"date\" : \"2021-08-18\",\n  \"rating\"
```

Comment:
@jmaslek  if I used VCR correctly, which according to this it seems I did, it shouldn't fail.

Reply:
my bad.  Didn't see you committed the new yaml

Reply:
I didn't explain myself properly I think.

But is still failing not sure why